### PR TITLE
added Community Forum

### LINKED ISSUE #19: [README] Need to add `Community Forum`
**Description**

<a href="https://github.com/layer5io/service-mesh-labs/blob/master/README.md">README.md</a> is the first file one should read when starting a new project. Our community launched `Discussion forum` for help seekers; needs to publish it officially on README.md for convenience.

### Screenshots

**Current behaviour:**
image

**Desired behaviour:**
image

**Note:**
Need to add only last line for `Community Forum`

**Environment:**
 - OS: any [e.g. Ubuntu]
 - Browser: any [e.g. Chrome, Safari]
 - Version: any [e.g. 22]
 - Device: any [e.g. laptop, iPhone 8]

---
**Reference:**
- Take reference to this <a href="https://github.com/layer5io/layer5/pull/1949">pull request</a>

### PR DESCRIPTION
**Pull request by gravishpurohit**
**Description**

This PR fixes #19

**Notes for Reviewers**

**[Signed commits](../CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Review by warunicorn19 — APPROVED** [review]
LGTM

**Comment by amino19** [comment]
@gravishpurohit : Pl fill the details <a href="https://github.com/layer5io/service-mesh-labs/pull/23#issue-693164070">here</a> & PR fixes #?
Mention issue number ie, `#19` in this case. :slightly_smiling_face:

**Comment by gravishpurohit** [comment]
Done @amino19 ,

File: README.md [hunk 1]
```diff
@@ -29,6 +29,7 @@ Find out more on the <a href="https://layer5.io/community">Layer5 community</a>.
 ✔️ <em><strong>Join</strong></em> any or all of the weekly meetings on <a href="https://calendar.google.com/calendar/b/1?cid=bGF5ZXI1LmlvX2VoMmFhOWRwZjFnNDBlbHZvYzc2MmpucGhzQGdyb3VwLmNhbGVuZGFyLmdvb2dsZS5jb20">community calendar</a>.<br />
 ✔️ <em><strong>Watch</strong></em> community <a href="https://www.youtube.com/playlist?list=PL3A-A6hPO2IMPPqVjuzgqNU5xwnFFn3n0">meeting recordings</a>.<br />
 ✔️ <em><strong>Access</strong></em> the <a href="https://drive.google.com/drive/u/4/folders/0ABH8aabN4WAKUk9PVA">community drive</a>.<br />
+✔️ <em><strong>Discuss</strong></em> in the <a href="https://discuss.layer5.io">Community Forum</a>.<br />
```
Comment (amino19) [inline README.md:32]: @gravishpurohit : LGTM! :partying_face:

**Review by amino19 — APPROVED** [review]

**Comment by amino19** [comment]
@navendu-pottekkat : Sir, at #19, there're 2 PRs, need to close one from Patil, as he's inactive due to some work, plus plus this fixed it successfully! :slightly_smiling_face:

**Review by leecalcote — APPROVED** [review]
Good deal. 👍

**Pull request merged**